### ISSUE TITLE
Investigate if we can update the ec2 tag 'Name' to use the 'guid' 

### ISSUE THREAD

**Issue by jwmatthews** [fusor/mig-agnosticd#13]
Currently when we look at the ec2 web console we see instances with names of:
'bastion', 'master1'.

I'd like to see if we can update the tag 'name' with '{guid}-{name}'

So for example, assume my "guid: jwm0710ocp4a" I  would want to see:
 'jwm0710ocp4a-bastion'
 'jwm0710ocp4a-master1'

**Comment by pranavgaikwad** [comment]
Yes we can do that. We will need to make a few changes in AgnosticD config.

Here's a PR I opened on my fork which adds an extra variable to specify a suffix to instance names - 

https://github.com/pranavgaikwad/agnosticd/pull/7
